spec/features/trainees/diversities/diversity_progress_spec.rb

```diff
@@ -0,0 +1,76 @@
+require "rails_helper"
+
+feature "completing the diversity section", type: :feature do
+  scenario "renders a 'not started' status when diversity details provided" do
+    given_a_new_trainee_is_created
+    and_i_visit_the_summary_page
+    then_the_diversity_section_should_be(Progress::STATUSES[:not_started])
```

Comment:
You could push `Progress::STATUSES[:not_started]` into the method and have something like:

```ruby
  def then_the_degree_status_should_be(status)
    expect(summary_page.degree_details.status.text).to eq(Progress::STATUSES[status])
  end
```

Then it becomes `then_the_diversity_section_should_be(:not_started)`

Reply:
done https://github.com/DFE-Digital/register-trainee-teachers/pull/171/commits/568772824da7ae4c3f3fee81e388b143d3d361f5